src/data/Tags.js

```diff
@@ -56,9 +56,6 @@ export const teamTags = [
   {
     text: "disqualify",
   },
-  {
```

Comment:
why is this here?

Reply:
Ah, I think I accidentally removed one of the tags from teamTags. I'll go fix it.